docker/tests/test-fedora-binaries.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import subprocess
+import sys
+
+binaries = [
```

Comment:
I slightly dislike the fact that binaries are now listed in multiple places (here, in the ubuntu script and implicitly in [packages.py](https://github.com/serokell/tezos-packaging/blob/master/docker/package/packages.py)) and it's easy to forget to update one of them once the set of binaries changes

Reply:
We can open a follow-up issue to hold JSON with all file names for example. I assume, that we require a full list of binaries in other places as well